certify-service/Dockerfile

```diff
@@ -55,13 +61,18 @@ ARG container_user_uid=1001
 ARG container_user_gid=1001
 
 
+ARG hsm_local_dir=hsm-client
+
+ENV hsm_local_dir_name=${hsm_local_dir}
+
 # install packages and create user
-RUN apk -q update \
-&& apk add -q unzip sudo bash curl \
-&& addgroup -g ${container_user_gid} ${container_user_group} \
-&& adduser -s /bin/sh -u ${container_user_uid} -G ${container_user_group} -h /home/${container_user} --disabled-password ${container_user}
+RUN apt-get -y update \
+&& apt-get install -y unzip sudo\
+&& groupadd -g ${container_user_gid} ${container_user_group} \
+&& useradd -u ${container_user_uid} -g ${container_user_group} -s /bin/sh -m ${container_user} \
+&& adduser ${container_user} sudo \
+&& echo "%sudo ALL=(ALL) NOPASSWD:/home/${container_user}/${hsm_local_dir}/install.sh" >> /etc/sudoers
```

Comment:
Are we creating a super user here, does it have a security impact ?

Reply:
this has been added for hsm client installation, taken it from [here](https://github.com/mosip/keymanager/blob/f26f2fa841a9ae1ee3595faa366c3cd18ed3044c/kernel/kernel-keymanager-service/Dockerfile#L84)

Reply:
I also had a separate chat with @Mahesh-Binayak on this, as per his comments, we are good.